include/aikido/planner/vectorfield/VectorFieldUtil.hpp

```diff
@@ -41,47 +34,83 @@ class DesiredTwistFunction : public dart::optimizer::Function
 
   /// Constructor.
   ///
-  /// \param[in] twist A desired twist
-  /// \param[in] jacobian System Jacobian
+  /// \param[in] twist A desired twist.
+  /// \param[in] jacobian System Jacobian.
   DesiredTwistFunction(const Twist& twist, const Jacobian& jacobian);
 
   /// Implementation inherited.
   /// Evaluating an objective by a state value.
   ///
-  /// \param[in] _qd Joint velocities
-  /// \return Objective value
-  double eval(const Eigen::VectorXd& _qd) override;
+  /// \param[in] qd Joint velocities.
+  /// \return Objective value.
+  double eval(const Eigen::VectorXd& qd) override;
 
   /// Implementation inherited.
   /// Evaluating gradient of an objective by a state value.
-  /// \param[in] _qd Joint velocities
-  /// \param[out] _grad Gradient of a defined objective
+  /// \param[in] qd Joint velocities.
+  /// \param[out] grad Gradient of a defined objective
   void evalGradient(
-      const Eigen::VectorXd& _qd, Eigen::Map<Eigen::VectorXd> _grad) override;
+      const Eigen::VectorXd& qd, Eigen::Map<Eigen::VectorXd> grad) override;
 
 private:
+  /// Twist.
   Twist mTwist;
+
+  /// Jacobian of Meta Skeleton
   Jacobian mJacobian;
 };
```

Comment:
I believe this is an implementation detail of `computeJointVelocityFromTwist()`. I suggest moving this class to an anonymous namespace in this `.cpp` file.